src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -937,3 +937,133 @@
     text: az monitor metrics list-definitions --resource /subscriptions/{subscriptionID}/resourceGroups/Space1999/providers/Microsoft.Network/networkSecurityGroups/ADDS-NSG
     crafted: true
 """
+
+helps['monitor log-alert'] = """
+type: group
+short-summary: Manage log alert rules.
+long-summary: >
+    See https://docs.microsoft.com/en-us/azure/azure-monitor/platform/alerts-unified-log for more information.
+"""
+
+helps['monitor log-alert create'] = """
+type: command
+short-summary: Create log alert rule using scheduledQueryRules API.
+long-summary: >
+    See https://docs.microsoft.com/en-us/rest/api/monitor/scheduledqueryrules/createorupdate for more information.
+parameters:
+  - name: --time-window
+    short-summary: Time window (in minutes) for which data needs to be fetched for query (should be greater than or equal to frequencyInMinutes).
+    long-summary: >
+      This argument overrules any time command (like ago) used in alert query.
+  - name: --alert-query
+    short-summary: Log search query. Required for action type - AlertingAction.
+    long-summary: >
+      Valid Azure Monitor log search query that will be executed at stated frequency with data for configured timeWindow.
+      See https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/log-query-overview for more information and examples.
+  - name: --query-type
+    long-summary: >
+      Currently only "ResultCount" is supported.
+  - name: --action-group
+    short-summary: Azure Action Group reference. Accepts space-separated action group identifiers. The identifier can be the action group's name
+      or its resource ID.
+  - name: --metric-column
+    short-summary: Evaluation of metric on a particular column.
+    long-summary: >
+      Specify the field to be used to aggregate-on and use as dimension for metric measurement log alert rule
+  - name: --metric-threshold
+    short-summary: The threshold of the metric trigger.
+    long-summary: >
+      Theshold is compared with numeric value in AggregatedValue field, used to trigger for metric measurement log alert rule.
+  - name: --metric-threshold-operator
+    short-summary: Result Condition Evaluation criteria
```

Comment:
Do we really need both short-summary and long-summary? The long-summary looks short too. Because, maybe in the future, we plan to deprecate the _help.py system. So it's better to put help information into the _param.py.